sdk/test-utils/multi-version/test/multiVersion.spec.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable no-use-before-define */
+import { assert } from "chai";
+import { isVersionInSupportedRange, versionsToTest } from "../src/multiVersion";
+
+describe.skip("Multi-service-version test support", () => {
+  const allVersions = ["1.0", "1.1", "1.2"];
+  describe("isVersionInSupportedRange() on version list", () => {
+    [
+      { currentVersion: "1.0", supported: ["1.0", "1.1"], expected: true },
+      { currentVersion: "1.2", supported: ["1.0", "1.1"], expected: false },
+      { currentVersion: "1.0", supported: { minVer: "1.0", maxVer: undefined }, expected: true },
+      { currentVersion: "1.1", supported: { minVer: "1.0", maxVer: undefined }, expected: true },
+      { currentVersion: "1.0", supported: { minVer: "1.1", maxVer: undefined }, expected: false },
+      { currentVersion: "1.0", supported: { minVer: undefined, maxVer: "1.1" }, expected: true },
+      { currentVersion: "1.1", supported: { minVer: undefined, maxVer: "1.1" }, expected: true },
+      { currentVersion: "1.2", supported: { minVer: undefined, maxVer: "1.1" }, expected: false }
+    ].forEach((arg) => {
+      const { currentVersion, supported, expected } = arg;
+      let versions: string;
+      if (supported instanceof Array) {
+        versions = `[${supported.join()}]`;
+      } else {
+        versions = `[min ${supported.minVer ?? "<unspecified>"}, max ${supported.maxVer ??
+          "<unspecified>"}]`;
+      }
+      it(`returns ${expected} for ${currentVersion} and supported versions ${versions}`, function() {
+        const result = isVersionInSupportedRange(currentVersion, supported, allVersions);
+        assert.equal(result.isSupported, expected);
+      });
+    });
+  });
+});
+
+const serviceVersions = ["7.0", "7.1"] as const;
+versionsToTest(serviceVersions, {}, (serviceVersion, onVersions) => {
```

Comment:
@ljian3377 do you think these scenarios are enough for Storage multi-version testing?

Reply:
LGTM. We still need a command line option to disable multi-version support when running live tests to allow us to save some time. We will need two sets of live tests, one only runs for the default version and the other runs for all specified versions.

Reply:
Right. We don't use command-line options but should be able to add another env variable to disable multi-version testing.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/12719/commits/3b6808288f1b156ac88825ecb2aa7ef83fb1fafc